apps/src/lib/util/AnalyticsConstants.js

```diff
@@ -175,12 +175,26 @@ const EVENTS = {
   SHARING_LINK_SEND_TO_PHONE:
     'User Clicks Send To Phone In Project Share Dialog',
   SHARING_CLOSE_ESCAPE: 'User Clicks X Or Esc Button In Project Share Dialog',
+
+  // Project sharing via 'Finish' button
+  FINISH_SHARING_LINK_COPY: 'User Clicks Project Copy Link In Share Dialog',
+  FINISH_SHARING_PUBLISH: 'User Clicks Publish In Project Share Dialog',
+  FINISH_SHARING_FB: 'User Clicks Facebook Icon In Project Share Dialog',
+  FINISH_SHARING_TWITTER: 'User Clicks Twitter Icon In Project Share Dialog',
+  FINISH_SHARING_LINK_SEND_TO_PHONE:
+    'User Clicks Send To Phone In Finish Congrats Dialog',
+  FINISH_BUTTON_CERTIFICATE:
+    'User Clicks on Finish Button in Finish Congrats Dialog - Certificate',
+
+  // Export app
+  EXPORT_APP: 'User Exports App From Share Advanced Options',
```

Comment:
It's not a requirement that events are in a group, right?

Reply:
Correct. I asked @samantha-code if grouping is helpful and she said 'Grouping events is super helpful especially when making charts on amplitude because I can go straight to that group rather than searching through all events!'